platforms/macos/src/node.rs

```diff
@@ -513,6 +538,240 @@ declare_class!(
             true
         }
 
+        #[sel(accessibilityNumberOfCharacters)]
+        fn number_of_characters(&self) -> NSInteger {
+            self.resolve(|node| {
+                if node.supports_text_ranges() {
+                    node.document_range().end().to_global_utf16_index() as _
```

Comment:
I'm unfamiliar with casting in rust, is this `_` similar to typescript's `any`?

Reply:
No, the `_` uses type inference to cast to whatever type is necessary. In this case, it's shorthand for casting to `NSInteger`.